Content.Server/Stack/Events/StackSplitAttemptEvent.cs

```diff
@@ -0,0 +1,33 @@
+namespace Content.Server.Stack.Events;
+
+public sealed class StackSplitAttemptEvent : CancellableEntityEventArgs
+{
+    /// <summary>
+    /// The item being Split by the user
+    /// </summary>
+    public EntityUid Used { get; }
+
+    /// <summary>
+    /// The User splitting the item
+    /// </summary>
+    public EntityUid User { get; }
+    public int Amount { get; }
+    public StackComponent? Stack { get; }
+    public TransformComponent? UserTransform { get; }
+
+    public StackSplitAttemptEvent
+    (
+        EntityUid used ,
+        EntityUid user ,
+        int amount ,
+        StackComponent? stack = null,
+        TransformComponent? userTransform = null
+    )
```

Comment:
what

Reply:
this is so fucked that GH syntax highlighting parsed it incorrectly. that is a sign that you should not do this

Reply:
I think(?) I fixed it